src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java

```diff
@@ -78,6 +85,7 @@
     set.addAll(searedRoad.values());
     set.addAll(searedTile.values());
   });
+  public static final RegistryObject<TileEntityType<TankTileEntity>> tank = TILE_ENTITIES.register("tank", TankTileEntity::new, (set) -> {set.addAll(searedTank.values());});
```

Comment:
Minor, but you can simplify to ` (set) -> set.addAll(searedTank.values())`

Reply:
Nice, I'll try to remember that one.

Reply:
Trick here is inline `() -> someFunction()` will return a value if it needs to, or return void if it needs to, based on the functional interface used (consumer or alike). The curly brackets clearly make it no return, but inline figures it out and looks cleaner.